public/controllers/management/components/management/ruleset/ruleset-editor.js

```diff
@@ -308,9 +306,10 @@ class WzRulesetEditor extends Component {
                         width="100%"
                         height={`calc(100vh - ${((showWarningRestart && !xmlError) || wazuhNotReadyYet) ? 300 : (xmlError ? (!showWarningRestart ? 245 : 350) : 230)}px)`}
                         value={content}
-                        onChange={newContent =>{
-                          this.setState({ content: newContent })}
-                        }
+                        onChange={this.setNewContent(newContent)}                        
+                        // onChange={newContent =>{
+                        //   this.setState({ content: newContent })}
```

Comment:
Great!! but pls remove this commend block :)

Reply:
done!